docs/l3/api_flow.md

```diff
@@ -44,8 +47,7 @@ Using `GET /members` on each node returns a list of all node aliases as set abov
   {
     "address": "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY",
     "alias": "Member A"
-  }
-]
+}
 ```
 # Match Creation API Flow (dscp-matchmaker-api)
```

Comment:
now there's links to the identity service swaggers, maybe want to add links to the matchmaker service swaggers here
```
- [MemberA](http://localhost:8000/swagger/)
- [MemberB](http://localhost:8010/swagger/)
- [Optimiser](http://localhost:8020/swagger/)
```

Reply:
Consider it done.